mkdocs/utils.py

```diff
@@ -35,6 +36,39 @@
     text_type = unicode         # noqa
 
 
+def yaml_load(source, loader=yaml.Loader):
+    """
+    Custom yaml loader.
+
+    Load all strings as unicode.
+    http://stackoverflow.com/a/2967461/3609487
+    """
+
+    class Loader(loader):
+
+        """Custom Loader."""
+
+        pass
```

Comment:
I'm really aware what pass is used for. It isn't required if a docstring is provided and pylint is absolteuly file with it. We already do this here: https://github.com/mkdocs/mkdocs/blob/master/mkdocs/exceptions.py